razorpay-sdk/src/Errors/ErrorCode.php

```diff
@@ -4,9 +4,18 @@
 
 class ErrorCode
 {
-    const BAD_REQUEST_ERROR                 = 'BAD_REQUEST_ERROR';
-    const SERVER_ERROR                      = 'SERVER_ERROR';
-    const GATEWAY_ERROR                     = 'GATEWAY_ERROR';
+    const BAD_REQUEST_ERROR                    = 'BAD_REQUEST_ERROR';
```

Comment:
Create a new class for this. Try not to make any changes to sdk that we can't push back upstream.